modules/migration/schemas_bindata.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+//go:build bindata
```

Comment:
Could this filename be changed bindata -> static

Reply:
It is meant to be consistent with the naming of all other bindata files ( BINDATA_DEST := modules/public/bindata.go modules/options/bindata.go modules/templates/bindata.go ) generators ( modules/public/public_bindata.go modules/options/options_bindata.go modules/templates/templates_bindata.go ) and not to be confused with the name of the file that contains the file modules/migration/schemas_static.go.

If that was already clear to you, I may have misunderstood your suggestion?

---
[source](https://lab.fedeproxy.eu/forgefriends/forgefriends/-/merge_requests/32#note_885)